src/shipit_code_coverage/shipit_code_coverage/notifier.py

```diff
@@ -45,8 +45,19 @@ def notify(self):
 
             try:
                 rev = changeset['node']
-                coverage = wait_until(lambda: self.get_coverage_summary(rev), 10)
-                if coverage is None:
+
+                class ResponseAcceptedError(Exception):
+                    pass
+
+                def get_coverage():
```

Comment:
I think this is not needed. If the result of get_coverage_summary is None, you `continue`, just like before (and as it is now, only in a more complex way).

Reply:
(Changing the get_coverage_summary to raise an exception instead of returning None in case of 202)

Reply:
Done in 0385197